bin/dfx-network-provider

```diff
@@ -20,13 +20,19 @@ source "$(clap.build)"
 
 export DFX_NETWORK
 
+get_home() {
+  eval echo "~$USER"
+}
```

Comment:
This gives an incorrect value if `dfx start` is run with HOME set to a different value from the one assumed by bash.  If there is an issue with HOME escaping a context to which it should be constrained, that problem should be fixed at source.

Reply:
I've added a TODO to remove this later.